### ISSUE TITLE
'rs' is empty object.

### ISSUE THREAD

**Issue by marklagendijk** [Sean-Der/WebSQL-For-FireFox#1]
Something seems to go wrong with exposing the properties of the result object:

``` javascript
if (aReason == 0)
{
    // TODO find out how to get the number of rows affected by the query
    var rs = { 'insertId':db.lastInsertRowID, 'rowsAffected':-1, 'rows':rows , __exposedProps__ : {rows : "r", rowsAffected : "r", insertId : "r"}};
    rs.rows.item = function(i){ return rows[i]; }
    rs.rows.length = rows.length;
    rs.rows.__exposedProps__ = {item : "r"};
    callbackFunc(self, rs);
}
```

Somehow the callback gets called with an empty result object. Do you have any idea what can be causing this?

**Comment by Sean-Der** [comment]
Hey Mark!

I forked this a little while ago, and am not really familiar with the code anymore. I will be able to work on this on the weekend though! If you get a chance to fix this before me, would you mind sending a pull request this way? I can't get the original authors to update their version.

**Comment by GarciaLozano** [comment]
Hi Mark & Sean!

In fact, there is something wrong in 'rs', but the error isn't properly in 'rs', It's in 'row' exposed props. I reviewed the code and I made some changes. With those changes it works fine! ;)

**Comment by Sean-Der** [comment]
This was fixed by 5a8a0497360ca0dfdead39869fba5edc7ec962a0, thanks  GarciaLozano
